x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/service/ElasticServiceAccounts.java

```diff
@@ -69,11 +69,14 @@ final class ElasticServiceAccounts {
                         "metrics-*",
                         "traces-*",
                         ".logs-endpoint.diagnostic.collection-*",
-                        ".logs-endpoint.action.responses-*",
-                        "profiling-*"
+                        ".logs-endpoint.action.responses-*"
                     )
                     .privileges("write", "create_index", "auto_configure")
                     .build(),
+                RoleDescriptor.IndicesPrivileges.builder()
+                    .indices("profiling-*")
+                    .privileges("read", "write", "create_index", "auto_configure")
```

Comment:
We don't need `create_index`, as the indices are managed by Fleet/Kibana

Reply:
I dropped it.